src/CabanaPD_Boundary.hpp

```diff
@@ -97,9 +149,7 @@ struct BoundaryIndexSpace<MemorySpace, RegionBoundary>
         Kokkos::RangePolicy<ExecSpace> policy( 0, particles.n_local );
         auto index_functor = KOKKOS_LAMBDA( const std::size_t pid )
         {
-            if ( x( pid, 0 ) >= plane.low_x && x( pid, 0 ) <= plane.high_x &&
-                 x( pid, 1 ) >= plane.low_y && x( pid, 1 ) <= plane.high_y &&
-                 x( pid, 2 ) >= plane.low_z && x( pid, 2 ) <= plane.high_z )
+            if ( region.compare( x, pid ) )
```

Comment:
What does this line do? Compare with what?

Reply:
Will change to `inside()`